BrokerApiNode/README.md

```diff
@@ -0,0 +1,82 @@
+# Broker APIs
+This example demostrates how to use the Streamr Broker API's from NodeJS. Node version 14/16 or higher is recommended.
+
+## Broker & SDKs
+- [Examples directory](https://github.com/streamr-dev/examples/blob/ECODR-34-broker-and-client/NodeJs/src/broker/)
```

Comment:
These links will probs break when this gets merged? Can you do a find + replace for `blob/ECODR-34-broker-and-client/`